src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DateValidators.java

```diff
@@ -62,4 +64,21 @@ private static boolean isDateOnOrAfterAnotherDate(LocalDate aDate,
 
         return true;
     }
+
+    public static boolean isCeasedDateOnOrAfterIndividualsDateOfBirth(LocalDate trustCeasedDate,
+                                                                           List<TrustIndividualDto> individuals,
+                                                                           String qualifiedFieldName,
+                                                                           Errors errors,
+                                                                           String loggingContext) {
+
+        for(TrustIndividualDto individual : individuals) {
+            if (trustCeasedDate.isBefore(individual.getDateOfBirth())) {
```

Comment:
I think you could save yourself a few lines of code here and simply call `isDateOnOrAfterAnotherDate()` - it's a method within this class. As soon as that method returns false, you'd want to return (essentially the same as you're doing now, I think).

Reply:
Refactored